TUTORIAL.md

```diff
@@ -1091,6 +1091,94 @@ of the type annotation, or even the function's implementation! The only way to h
 Similarly, the only way to have a function whose type is `a -> a` is if the function's implementation returns
 its argument without modifying it in any way. This is known as [the identity function](https://en.wikipedia.org/wiki/Identity_function).
 
+## Tag Unions
+
+We can also annotate types that include tags:
+
+```coffee
+colorFromStr : Str -> [Red, Green, Yellow]
+colorFromStr = \string ->
+    when string is
+        "red" -> Red
+        "green" -> Green
+        _ -> Yellow
+```
+
+You can read the type `[Red, Green, Yellow]` as "a *tag union* of the tags `Red`, `Green`, and `Yellow`."
+
+Tags are always annotated as being part of a tag union, even if that union only has one tag in it. For example:
```

Comment:
I found this a bit confusing to read, since tags don't *have* to be annotated, and "tags" seems to be overloaded in both type and value - what about something like
```suggestion
Tags are always part of a tag union type, even if that union only has one tag in it. For example:
```